packages/platforms/mellanox/x86-64/x86-64-mlnx-msn2740/onlp/builds/src/module/inc/x86_64_mlnx_msn2740/x86_64_mlnx_msn2740_config.h

```diff
@@ -0,0 +1,137 @@
+/**************************************************************************//**
+ *
+ * @file
+ * @brief x86_64_mlnx_msn2740 Configuration Header
+ *
+ * @addtogroup x86_64_mlnx_msn2740-config
+ * @{
+ *
+ *****************************************************************************/
+#ifndef __x86_64_mlnx_msn2740_CONFIG_H__
```

Comment:
Please re-run the autogenerator. It will update all of these symbols to uppercase. 
s/x86_64_mlnx_msn2740/X86_64_MLNX_MSN2740/g

Reply:
pulling back request and issuing new one with comments.